src/views/Layout.jsx

```diff
@@ -21,55 +38,72 @@ export function Layout({ listToken }) {
 					minHeight: '100%',
 				}}
 			>
+				{/* App Bar begins here */}
 				<Container maxWidth="xl" sx={{ p: '0' }} disableGutters>
 					<AppBar position="relative">
 						<Container maxWidth="xl">
-							<Toolbar sx={{ justifyContent: 'space-between' }}>
-								<Typography
-									variant="h5"
-									noWrap
-									component="a"
-									href="/"
-									sx={{
-										mr: 2,
-										display: { xs: 'none', md: 'flex' },
-										fontFamily: 'monospace',
-										fontWeight: 700,
-										letterSpacing: '.3rem',
-										color: 'inherit',
-										textDecoration: 'none',
-									}}
-								>
-									Smart Shopping List
-								</Typography>
+							<Toolbar disableGutters>
+								{/* This is the App Icon Menu */}
 								{listToken ? (
-									<div>
-										{/* Consider using drawer or nav icon for mobile screen size */}
-										<Breadcrumbs
-											aria-label="breadcrumb"
-											color="white"
-											separator=""
+									<Box
+										sx={{ flexGrow: 1, display: { xs: 'flex', md: 'none' } }}
+									>
+										<IconButton
+											size="large"
+											aria-label="account of current user"
+											aria-controls="menu-appbar"
+											aria-haspopup="true"
+											onClick={handleOpenNavMenu}
+											color="inherit"
+										>
+											<MenuIcon sx={{ mr: 1 }} />
+										</IconButton>
+
+										{/* This is the menu displayed after clicking on the icon */}
+										<Menu
+											id="menu-appbar"
+											anchorEl={anchorElNav}
+											anchorOrigin={{
+												vertical: 'bottom',
+												horizontal: 'left',
+											}}
+											keepMounted
+											transformOrigin={{
+												vertical: 'top',
+												horizontal: 'left',
+											}}
+											open={Boolean(anchorElNav)}
+											onClose={handleCloseNavMenu}
+											sx={{
+												display: { xs: 'block', md: 'none' },
+											}}
 										>
-											<NavLink to="/list" className="Nav-link">
-												List
-											</NavLink>
-											<NavLink to="/add-item" className="Nav-link">
-												Add Item
-											</NavLink>
-										</Breadcrumbs>
-									</div>
+											<MenuItem onClick={handleCloseNavMenu}>
+												<NavLink to="/list" className="Menu-nav-link">
+													List
+												</NavLink>
+											</MenuItem>
+
+											<MenuItem onClick={handleCloseNavMenu}>
+												<NavLink to="/add-item" className="Menu-nav-link">
+													Add Item
+												</NavLink>
+											</MenuItem>
+										</Menu>
+									</Box>
 								) : (
-									<div></div>
+									<Box></Box>
 								)}
 
+								{/* This is the title displayed on the left for medium to large screen sizes */}
+								<BreakfastDiningOutlinedIcon sx={{ mr: 1 }} />
```

Comment:
I think the toast icon adds some fun character but could also be a bit misleading to the user as it kind of resembles a button or some kind of interactive element. I think the nav bar might be stronger without it. Just my two cents!